app/lib/dartdoc/models.dart

```diff
@@ -16,23 +16,58 @@ import 'storage_path.dart' as storage_path;
 
 part 'models.g.dart';
 
+/// Describes the details of a dartdoc-generated content.
 @JsonSerializable()
 class DartdocEntry {
+  /// Unique identifier of the entry, generated via a random UUID.
   final String uuid;
+
+  /// The package name.
   final String packageName;
+
+  /// The package version.
   final String packageVersion;
+
+  /// Whether the [packageVersion] is the latest stable version of the package
+  /// (at the time of the entry being created, but may be updated later).
   final bool isLatest;
+
+  /// Whether the package version is too old. This is never set if the version
+  /// is the latest stable version of the package..
   final bool isObsolete;
+
+  /// Whether the package version uses Flutter.
   final bool usesFlutter;
+
+  /// The pub site runtime version of the runtime that generated the content.
   final String runtimeVersion;
+
+  /// The SDK version that was used to fetch dependencies.
   final String sdkVersion;
+
+  /// The version of `package:dartdoc` that generated the content.
   final String dartdocVersion;
+
+  /// The version of Flutter that was used to fetch dependencies.
   final String flutterVersion;
+
+  /// The version of pub site customization over the dartdoc-generated HTML.
+  /// See [versions.customizationVersion].
   final String customizationVersion;
+
+  /// When the content was generated.
   final DateTime timestamp;
```

Comment:
For future reference, maybe this is more relevant in history where we still can make changes (maybe)...

But we should have named this field `created` and then we wouldn't need a documentation comment :)

Reply:
Shall I add a new `created` field and do some backward-compatible migration?

Reply:
No, it was just a note for the future :)